assets/blocks/course-outline/course-block/edit.js

```diff
@@ -74,26 +81,29 @@ const EditCourseOutlineBlock = ( {
 
 	return (
 		<>
-			<OutlineBlockSettings
-				animationsEnabled={ attributes.animationsEnabled }
-				setAnimationsEnabled={ updateAnimationsEnabled }
-			/>
-
-			<section className={ className }>
-				<OutlineAttributesContext.Provider
-					value={ {
-						outlineAttributes: attributes,
-						outlineSetAttributes: setAttributes,
-					} }
+			<OutlineAttributesContext.Provider
+				value={ {
+					outlineAttributes: attributes,
+					outlineSetAttributes: setAttributes,
```

Comment:
I checked that we're not using the `outlineSetAttributes` in any component, only the `outlineAttributes`. Did we already add it for a future feature?

Also, I think it's not part of this PR, but instead using the context directly, we could use the [block context](https://developer.wordpress.org/block-editor/developers/block-api/block-context/) for that. WDYT?

Reply:
That would be a good fit, but we likely want to use the setter too, and block context doesn't have that. Yeah the set was there for the future I think, using it for an upcoming shared settings PR.